src/test/java/org/jpeek/graph/XmlGraphTest.java

```diff
@@ -32,54 +31,51 @@
 import org.hamcrest.core.AllOf;
 import org.jpeek.FakeBase;
 import org.jpeek.skeleton.Skeleton;
-import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 import org.llorllale.cactoos.matchers.Assertion;
 import org.llorllale.cactoos.matchers.HasValues;
 import org.llorllale.cactoos.matchers.HasValuesMatching;
 
 /**
  * Test case for {@link XmlGraph}.
  * @since 0.30.9
- * @todo #413:30min In #413 we evolved graph connection building to take into account overloaded
- *  method differentiation as stated in #403. Wait until #403 is fully resolved (for now, it still
- *  has the puzzle #437) and then activate back the 2 tests in this class.
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
-public final class XmlGraphTest {
+final class XmlGraphTest {
 
     /**
      * First method name.
      */
-    private static final String METHOD_ONE = "MethodMethodCalls.methodOne";
+    private static final String METHOD_ONE = "MethodMethodCalls.methodOne.";
 
     /**
      * Second method name.
      */
-    private static final String METHOD_TWO = "MethodMethodCalls.methodTwo";
+    private static final String METHOD_TWO = "MethodMethodCalls.methodTwo.";
 
     /**
      * Third method name.
      */
-    private static final String METHOD_THREE = "MethodMethodCalls.methodThree";
+    private static final String METHOD_THREE = "MethodMethodCalls.methodThree.";
 
     /**
      * Fourth method name.
      */
-    private static final String METHOD_FOUR = "MethodMethodCalls.methodFour";
+    private static final String METHOD_FOUR = "MethodMethodCalls.methodFour.";
 
     /**
      * Fifth method name.
      */
-    private static final String METHOD_FIVE = "MethodMethodCalls.methodFive";
+    private static final String METHOD_FIVE = "MethodMethodCalls.methodFive.";
```

Comment:
@vzurauskas can you explain why is it expected that the method ends up with a dot?

Reply:
@victornoel These strings are names of nodes. A node name currently is the signature of the method it represents - `XmlMethodSignature`. `XmlMethodSignature` is implemented such that it always has a dot after the method name, even if it has no arguments:

```java
new Joined(
    "", call.xpath("op/name/text()").get(0),
    ".", new XmlMethodArgs(call.nodes("op").get(0)).toString()
)
```

Reply:
@vzurauskas Can we change that? it is a too surprising behaviour IMHO.

Reply:
@victornoel I'm not sure. It's used only in the internal implementation of `graph` package, so nobody outside it will be surprised by it. And removing the dot would involve a procedural `if (part of evaluated element is empty) remove trailing dot;`, which would ruin the elegant declarative style which currently involves only compositions of `Text`s (or `String`s in some cases) in the whole package.

Reply:
@vzurauskas ok, good for me then :)